.github/workflows/appflowy_editor_test.yml

```diff
@@ -0,0 +1,52 @@
+name: AppFlowyEditor test
```

Comment:
Don't add this file back. We have moved it to the `appflowy_editor` repo

Reply:
Deleted